src/attributes.js

```diff
@@ -14,7 +14,56 @@
  * limitations under the License.
  */
 
-var getData = require('./node_data').getData;
+var nodeData = require('./node_data'),
+    getAttrsArr = nodeData.getAttrsArr,
+    getNewAttrs = nodeData.getNewAttrs,
+    getAttrs = nodeData.getAttrs;
+
+
+/**
+ * The offset in the virtual element declaration where the attributes are
+ * specified.
+ * @const {number}
+ */
+var ATTRIBUTES_OFFSET_INTERNAL = 3;
+
+
+/**
+ * The offset in the update attributes call where the attributes are specified.
+ * @const {number}
+ */
+var ATTRIBUTES_OFFSET_EXTERNAL = 1;
+
+
+/**
+ * Verify if the script are running in production.
```

Comment:
Fixed it in both this and the [`virtual_elements` module](https://github.com/google/incremental-dom/blob/master/src/virtual_elements.js#L44).